Repairs/app/src/main/res/layout/activity_dashboard.xml

```diff
@@ -506,7 +506,7 @@
             android:layout_marginTop="29dp"
             android:layout_marginEnd="124dp"
             android:layout_marginBottom="49dp"
-            android:text="@string/three"
+            android:text="0"
```

Comment:
text in the xml files should come from the string resource file, not hardcoded strings